### PR TITLE
Feature/add jellyfish sales order payone

### PR DESCRIPTION
**Pull request by SpiderDan98**

### CODE REVIEW

File: bundles/jellyfish-sales-order-payone/composer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "description": "JellyfishSalesOrderPayone module.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.2",
```
Comment (daniel-rose) [inline bundles/jellyfish-sales-order-payone/composer.json:12]: 7.2 EOL. Please use `>=7.3`

File: bundles/jellyfish-sales-order-payone/composer.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "fond-of-oryx/jellyfish-sales-order": "^1.0.0",
+    "spryker-eco/payone": "^4.0.0",
+    "spryker/sales": "^11.0.0"
```
Comment (daniel-rose) [inline bundles/jellyfish-sales-order-payone/composer.json:15]: Please check other sales versions.

File: bundles/jellyfish-sales-order-payone/composer.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.2",
+    "fond-of-oryx/jellyfish-sales-order": "^1.0.0",
+    "spryker-eco/payone": "^4.0.0",
```
Comment (daniel-rose) [inline bundles/jellyfish-sales-order-payone/composer.json:14]: Please check other payone versions.

**Review by gengjozsef — APPROVED** [review]

File: bundles/jellyfish-sales-order-payone/composer.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.2",
+    "fond-of-oryx/jellyfish-sales-order": "^1.0.0",
```
Comment (daniel-rose) [inline bundles/jellyfish-sales-order-payone/composer.json:13]: Are you sure, that you need this package. Maybe jellyfish-sales-order-extension would be better.

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**